superset-frontend/src/assets/images/icons/filter.svg

```diff
@@ -16,6 +16,7 @@ KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
 -->
-<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think we already have this icon - it's called `filter_small.svg`

Reply:
The filter_small icon has a lot of padding around and it does not respect the sizes being given through the Icons component. In order to get a size M you would need to give it a XL. I believe the reason for that is because indeed it was meant to be "small". For that reason, I replaced the actual filter icon with the one from our design files as I believe it should be the same everywhere

Reply:
Got it 👍